frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -182,7 +201,18 @@ export const personsLogic = kea<personsLogicType<PersonPaginatedResponse>>({
                 actions.loadPersons()
             }
         },
-        '/person/*': ({ _ }: { _: string }) => {
+        '/person/*': (
+            {
+                _,
+            }: {
+                _: string
+            },
+            _searchParams: Record<string, string>,
+            { activeTab }: Record<string, any>
```

Comment:
I made some improvements to types in kea and these don't need to be manually specified anymore.

Reply:
@mariusandra, not too sure what you mean here. Could you explain what I should take out here?

Reply:
Some TS errors are now reported on this logic.

Basically there's no need to specify any types in this line. So it could be just:
```
        '/person/*': ({ _: person }, _searchParams, { activeTab }) => { ... }
```

Reply:
I think my kea version in `node_modules` was stale or something because typescript kept complaining when I took these types out. Note to self - clear `node_modules` and run `yarn install` when dependencies are updated